.yamato/pytest-gpu.yml

```diff
@@ -15,9 +15,11 @@ pytest_gpu:
       python3 -m pytest -m "not check_environment_trains" --junitxml=junit/test-results.xml -p no:warnings
   triggers:
     cancel_old_ci: true
-    recurring:
-      - branch: main
-        frequency: daily
+    expression: |
+      (push.branch eq "main" OR
```

Comment:
could you create a branch off of this branch, update the triggers to be this branch, then merge it into this branch to see if they trigger?

Reply:
I'd like to train an agent that understand this sentence.

See #5132 #5135. GPU test is triggered.

Reply:
badabing!